### DISCUSSION TITLE
How can I add citations inside a default table (or alternative tablex)?

### DISCUSSION THREAD

**Discussion by linhduongtuan** [typst/typst#1480] [Q&A]
Dear folks,
Do you know how can add citations inside a default table (or alternative tablex)?
Moreover, how can I set a list of tables using tablex?
Thank in advance.
Linh

**Comment by Dherse** [comment]
You can wrap your tables in figures and add the `kind: table` to your figure to make it the correct type :)

Here is an example:

```ts
figure(caption: [ My awesome caption! ], kind: table)[
  #table(
    columns: 2,
    [ Hello, world! ],
    [ Hello, linhduongtuan! ],
  )
]
```

More info on the figure doc: https://typst.app/docs/reference/meta/figure/

Hope this helps :)
